packages/ui/src/viewer-table/InteractionPanel/DictionaryDownloadButton.tsx

```diff
@@ -70,11 +79,17 @@ const DictionaryDownloadButton = ({ fileType, iconOnly = false }: DictionaryDown
 
 	const { name, version } = selectedDictionary;
 
-	const fetchUrl = `${lecternUrl}/dictionaries/template/download?${new URLSearchParams({
+	const queryParams = new URLSearchParams({
 		name,
 		version,
 		fileType,
-	})}`;
+	});
+
+	if (schemaName !== undefined && schemaName.trim()) {
```

Comment:
`trim` doesn't mutate the original, returns a copy, so checking in this conditional doesn't actually trim it to append in queryParams